.travis.yml

```diff
@@ -54,9 +65,13 @@ before_script:
   - php -r "echo INTL_ICU_DATA_VERSION . \"\n\";"
   - mysql --version
   - psql --version
+  - node --version
+  - npm --version
 
   # initialize databases
   - mysql -e 'CREATE DATABASE yiitest;';
+  - mysql -e "CREATE USER 'yiitest'@'localhost' IDENTIFIED BY 'yiitest'";
```

Comment:
Using `root` user fails:

```
yii\db\Exception: SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: NO)
```
Using `travis` user fails too:

```
yii\db\Exception: SQLSTATE[HY000] [1045] Access denied for user 'travis'@'localhost' (using password: NO)
```

See also #12971.

So I decided to create separate dedicated user for test.

Also note that trusty by default uses MySQL 5.6, while precise - 5.5.